content/blog/2020/07/2020-07-27-repository-signing-keys-changing.adoc

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: "Jenkins 2.235.3: New Linux Repository Signing Keys"
+tags:
+- announcement
+- linux
+- platform
+author: markewaite
+description: >
+    This article announces new Linux repository signing keys.
+    Repository signing keys will change with the release of Jenkins LTS 2.235.3.
+    The previous repository signing keys will not be used after the release of Jenkins LTS 2.235.3.
+    The repository signing keys will be changed for Debian/Ubuntu and CentOS/Red Hat repositories.
+opengraph:
+  image: /images/post-images/2020/07-repository-signing-keys-change/opengraph.png
+---
+
+The Jenkins core release automation project has been delivering Jenkins weekly releases since link:/changelog/#v2.232[Jenkins 2.232], April 16, 2020.
+The Linux repositories that deliver the weekly release were updated with new GPG keys with the release of Jenkins 2.232.
+
+Beginning with Jenkins LTS release link:/changelog-stable/#v2.235.3[2.235.3], stable repositories will be signed with the same GPG keys that sign the weekly repositories.
+Administrators of Linux systems *must* install the new signing keys on their Linux servers *before* installing Jenkins 2.235.3.
+
+== Debian/Ubuntu
+
+Update Debian compatible operating systems (Debian, Ubuntu, Linux Mint Debian Edition, etc.) with the command:
+
+.Debian
+----
+# wget -qO - https://pkg.jenkins.io/debian-stable/jenkins.io.key | apt-key add -
+----
+
+== Red Hat/CentOS
+
+Update Red Hat compatible operating systems (Red Hat Enterprise Linux, CentOS, Fedora, Oracle Linux, Scientific Linux, etc.) with the command:
+
+.Red Hat
+----
+# rpm --import https://pkg.jenkins.io/redhat-stable/jenkins.io.key
```

Comment:
Is this key being updated ahead of time or will the blog not go out until just after the release?

Reply:
I had intended to publish the blog at the same time as the new release because the new signing key won't be available at the referenced location until the release has been uploaded.